framework/src/main/java/org/tron/core/services/ratelimiter/RateLimiterInterceptor.java

```diff
@@ -90,6 +95,17 @@ public void init(Server server) {
   public <ReqT, RespT> Listener<ReqT> interceptCall(ServerCall<ReqT, RespT> call, Metadata headers,
       ServerCallHandler<ReqT, RespT> next) {
 
+
+    String methodMeterName = MetricsKey.NET_API_DETAIL_ENDPOINT_QPS
```

Comment:
Why does this interface need to be handled separately

Reply:
good. will use metricUtil new interface to handle those issue